linera-service/src/linera.rs

```diff
@@ -1633,6 +1644,49 @@ async fn main() -> Result<(), anyhow::Error> {
             Ok(())
         }
 
+        ClientCommand::Net(net_command) => match net_command {
+            NetCommand::Up => {
+                let network = Network::Grpc;
+                let mut net = LocalNetwork::new(network, 1)?;
+                let client1 = net.make_client(network);
+                let client2 = net.make_client(network);
+
+                net.generate_initial_validator_config().await?;
+                client1.create_genesis_config().await?;
+                client2.wallet_init(&[]).await?;
+
+                // Create initial server and client config.
+                net.run_local_net().await?;
+                let net_path = net.net_path();
+
+                println!(
+                    "\nLinera net directory available at: {}",
+                    net_path.display()
+                );
+                println!("To configure your Linera client for this network, run:\n");
```

Comment:
Maybe "log" the unstructured text (or equivalently print to stderr) so that stdout can be piped into bash?

Reply:
Good idea :)

Reply:
I'm going to go with stderr as logging prefixes the output making it harder to copy/paste into a new terminal.